web-ui/src/common/util.js

```diff
@@ -1,10 +1,16 @@
+import queryString from 'query-string';
 import browser from 'helpers/browser';
 
 export const hasQueryParameter = (param) => {
   const decodedUri = decodeURIComponent(window.location.search.substring(1));
   return !(decodedUri.split('&').indexOf(param) < 0);
 };
```

Comment:
These observations are not really from this PR, but does this method still need these handmade parsing after adding that js library to parse querystring?